test/tests/rails3.rb

```diff
@@ -694,7 +704,7 @@ def test_sql_injection_in_template
     #SQL injection in controllers should not warn again in views
     assert_no_warning :type => :template,
       :warning_type => "SQL Injection",
-      :line => 3, #This should be line 4 :(
```

Comment:
I changed this one, since it looked like it would have been caused by the same issue. But, I wasn't 100% sure, since this test passed either way, given it is a `assert_no_warning`.